editor/code/lib/types.ts

```diff
@@ -113,12 +114,22 @@ export interface FlecheSaveParams {
   textDocument: VersionedTextDocumentIdentifier;
 }
 
-export interface SentencePerfParams {
-  loc: Loc;
+export interface PerfInfo {
+  // Original Execution Time (when not cached)
   time: number;
+  // Whether the execution was cached
+  cached: boolean;
+  // "Real" Execution Time, often very low for memoized sentences
+  real: number;
+  // Difference in words allocated in the heap using `Gc.quick_stat`
   mem: number;
 }
 
+export interface SentencePerfParams {
+  range: Range;
```

Comment:
This is not the correct range I believe. It should be vscode.Range.

Reply:
This is part of the protocol so it should use the protocol types no? What would other clients do?

On top of that vscode ranges may not properly serialize

Reply:
Alright that makes sense. I did the other change in my PR, but I didn't realise that the serializability was an issue. There should be a standard conversion function between the two types right? I couldn't find it.